docs/OpenCLCOnVulkan.md

```diff
@@ -203,16 +211,30 @@ The shaders produced use the GLSL450 memory model. As such, there is an assumpti
 no aliasing by default. The compiler does not generate *Aliased* decorations
 currently. Users should be aware of this and ensure they are not relying on aliasing.
 
-#### Descriptor map
+#### Embedded Reflection Instructions
+
+Clspv embeds reflection information via use of a non-semantic extended
+instruction set. The instruction set is defined
+[here](http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/nonsemantic/NonSemantic.ClspvReflection.html).
```

Comment:
I would recommend actually naming the extended instruction set in the visible words, so only reading the text tells me what it is. That supports and promotes the extended instruction set itself.

> Clspv embeds reflection information via use of the `[NonSemantic.ClspvReflection.0](http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/nonsemantic/NonSemantic.ClspvReflection.html)` non-semantic extended
instruction set.